torch/linalg/__init__.py

```diff
@@ -953,6 +1076,84 @@
     tensor(5.4345)
 """)
 
+matrix_norm = _add_docstr(_linalg.linalg_matrix_norm, r"""
+linalg.matrix_norm(A, ord=None, dim=None, keepdim=False, *, dtype=None, out=None) -> Tensor
+
+Computes a matrix norm.
+
+If :attr:`A` is complex valued, it computes the norm of :attr:`A`\ `.abs()`
+
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
+
+- If :attr:`dim`\ `= None`, the norm will be computed over the last two dimensions
+  and the other dimensions will be treated as batch dimensions.
+- If :attr:`dim` is a `2`-`tuple`, the norm will be computed over these two dimensions
+  and the other dimensions will be treated as batch dimensions.
+
+:attr:`ord` defines the matrix norm that is computed. The following norms are supported:
+
+======================   ========================================================
+:attr:`ord`              matrix norm
+======================   ========================================================
+`None` (default)         Frobenius norm
```

Comment:
@Lezcano @IvanYashchuk @nikitaved @kurtamohler  I think some of us talked about this very briefly. Should the default for norms be 'fro' instead of `None`? 'fro' would actually specify the default behavior

Reply:
I think we should (and same for `vector_norm`).

Reply:
I agree,`fro` for default!

Reply:
I agree with this proposal and changed the default to 'fro'.